components/brave_rewards/resources/page/components/promotion.tsx

```diff
@@ -171,10 +138,21 @@ class Grant extends React.Component<Props, State> {
     )
   }
 
+  convertPromotionTypesToType = (type: Rewards.PromotionTypes) => {
+    switch (type) {
+      case 1: { // Rewards.PromotionTypes.ADS
```

Comment:
nit: TODO so a const doesn't get forgotten

Reply:
right now we can't use direct definition so I added reference for a number